File: src/main/java/com/itsci/mju/maebanjumpen/partyrole/repository/AccountManagerRepository.java
package com.itsci.mju.maebanjumpen.partyrole.repository;

import com.itsci.mju.maebanjumpen.entity.AccountManager;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface AccountManagerRepository extends JpaRepository<AccountManager, Integer> {
}

File: src/main/kotlin/com/itsci/mju/maebanjumpen/partyrole/repository/AccountManagerRepository.kt
package com.itsci.mju.maebanjumpen.partyrole.repository

import com.itsci.mju.maebanjumpen.entity.AccountManager
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface AccountManagerRepository : JpaRepository<AccountManager, Int>

